### ISSUE TITLE
Ampache installation fails at Step 2

### ISSUE THREAD

**Issue by headrift** [ampache/ampache#3779]
## Description

I'm having difficulties getting Ampache installed. If I don't initialize the database it gives me a test.php page that looks good expect for the error that the database tables were not found. If I do initialize the database by:

mariadb -u root -p
>>> create database ampache;
*** I still get the database tables error at this point
>>> use ampache;
>>> grant all on ampache to 'ampache'@'localhost' identified by '********';
>>> grant all on ampache.* to 'ampache'@'localhost' identified by '********';
*** still no love from the database check

So I do the next logical thing:

sudo mariadb -u ampache -p ampache </usr/share/webapps/ampache/resources/sql/ampache.sql

...which takes a minute or 5, but gets me to a more or less blank Ampache page if I refresh ampache/ -- so, carrying on, I open install.php, which gets to the the select a language, and then Step 1 where I put in the correct info, check the box for "overwrite database", add the database user "ampache" with the password I'm using. if I click "install database" it gives me an error about root's access being denied, but if I fill it out and press enter it goes to Step 2.

From here, everything about the user is filled out, and I put it up to "Community", ffmpeg, uncheck Subsonic, check UPNP and WebDAV. "File Insight" looks good, all green.

The I click "create config" and it takes me to a blank page at ampache/install.php?action=create_config with nothing in the file. Pressing enter throws me an htaccess file similar to the ones I already have in ampache/play and ampache/rest, but without the leading . -- fixing all those things gets me nowhere further.

### Describe the bug

The install.php does not work

### To reproduce

^ see above

### Expected behavior

It should be installed

## Environment

* Ampache version: 6.0.3-1
* Web server + version: Apache 2.4.58-1
* Server operating system: Arch Linux
* Client operating system: Arch Linux / Chrome

### Client type

* Web
* Ampache
* UPnP
* WebDAV

## Logs

*** Apache log on refresh of blank page

_gateway - - - [21/Jan/2024:00:39:20 +0000] "POST /ampache/install.php?action=create_config HTTP/1.1" 200 - "https://thechatpit.com/ampache/install.php?action=create_db&htmllang=en_US&charset=UTF-8" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36"

Anyway, hope this helps. I had to rebuild my server a month or two ago and am just getting to Ampache, which I've had before and liked. There are ampache and ampache-git in the repo, but I'm trying to use ampache without git pulls because it says it's less out of date *shrug*

**Comment by headrift** [comment]
I'm now using Ampache 8.3, same problem

**Comment by lachlan-00** [comment]
You can install from install.php, this web setup creates the database, tables and users for you as well as creates the config. 

If you have precreated the database and database user you need to fiddle around with the settings a bit, i find its easier to delete the database and users if starting from scratch and then going through the web setup.

if you want to do it manually these are the commands i use to test the install zips

## Forcibly install the database
```
php /var/www/html/bin/installer install -f -U root -P $ROOTPASSWORD -H $SQLIP -u $AMPACHEDATABASEUSER -p $AMPACHEDATABASEPASSWORD -d $DATABASENAME
```
you don't need to do this if you install the sql file

## Now you have a database you need an admin user
```
php /var/www/html/bin/cli admin:addUser admin -p $ADMINPASSWORD -e admin@ampache.dev -l 100
```

## Then you need to create the config file

so copy `config/ampache.cfg.php.dist` to `config/ampache.cfg.php` and edit the DB fields

![image](https://github.com/ampache/ampache/assets/1305249/ceead798-2b05-4e6e-a28f-9bef8a82701f)

## TLDR if manually installing

* Create tables using `ampache.sql`
* Add admin user using `bin/installer`
* Create config file `config/ampache.cfg.php`

OPnce you have a database with tables. an admin user and a config file you should be able to log in

**Comment by headrift** [comment]
Okay, sorry, must not have been descriptive enough with what I've done. I'll run through it again while I'm typing this so I hopefully don't miss anything.

Step 1, logging in to the database system:

$ sudo mariadb -u root -p

There's an ampache database, so I'll drop it:

mariadb> drop database ampache;

Now I'll get one set back up:

mariadb> create database ampache;
mariadb> use ampache
mariadb> grant all on ampache to 'ampache'@'localhost' identified by 'password';
mariadb> grant all on ampache.* to 'ampache'@'localhost' identified by 'password';

Then I throw in what looks like the base db:

$ sudo mariadb -u ampache -p ampache </usr/share/webapps/ampache/resources/sql/ampache.sql

I add a couple users with the suggestion above:

$ /usr/share/webapps/ampache/bin/cli admin:addUser admin -p '********' -l 100
$ /usr/share/webapps/ampache/bin/cli admin:addUser user1 -p '********' -l 100

Problem happens there, and I get these errors:

PHP Warning:  ini_set(): Session ini settings cannot be changed when a session is active in /usr/share/webapps/ampache/src/Module/System/Session.php on line 627
PHP Warning:  session_set_save_handler(): Session save handler cannot be changed when a session is active in /usr/share/webapps/ampache/src/Module/System/Session.php on line 629

When I got back to try install.php (or going to ampache/) I get an empty file back. So basically no change. If I drop the db again and try using install.php instead I get the same problems
